package.json

```diff
@@ -17,6 +17,10 @@
     "*.ts": [
       "prettier --parser typescript --write",
       "git add"
+    ],
+    "*.md": [
```

Comment:
I remembered that prettier can reformat Markdown, so since I was editing the README anyway I figured I'd add that here.